lms/djangoapps/mobile_api/testutils.py

```diff
@@ -40,6 +61,8 @@ def setUp(self):
         self.password = 'test'
         self.username = self.user.username
 
+        self.prereq_course = CourseFactory.create()
```

Comment:
But it can still be defined within the appropriate context of `_add_prerequisite_course`.